Mesh4all/m4a-firmware
# shell extended tool: extends shell funct. to our custom modules

### Contribution description

Extends shell functionality of some modules
 
### Testing procedure

An example of application is; `examples/shell_extended`, how it works is described as usual in the documentation, flash and open a terminal, type help, it should display a list of available commands.

### Issues/PRs references
None
